gensim/models/word2vec.py

```diff
@@ -376,16 +376,17 @@ def __init__(
         need about 1GB of RAM. Set to `None` for no limit (default).
 
         `sample` = threshold for configuring which higher-frequency words are randomly downsampled;
-            default is 0 (off), useful value is 1e-5.
+            default is 1e-3, useful value is 1e-5, 0 stands for off.
 
         `workers` = use this many worker threads to train the model (=faster training with multicore machines).
 
-        `hs` = if 1 (default), hierarchical sampling will be used for model training (else set to 0).
+        `hs` = if 1 (default), hierarchical sampling will be used for model training (if set to 0, negative sampling will be used).
```

Comment:
Technically `hs=0` is not enough to trigger the use of negative-sampling: `negative` must be set non-zero as well. (Also: line-length.)